### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/openrc/commit/be59c737a9dd4e4be5d547c5114c752224fe2c80#commitcomment-71364823

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/openrc/commit/f0cdc948a21b758963b826bb43959684c333e845

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: conf.d/keymaps [hunk 1]
```diff
@@ -2,11 +2,11 @@
 # of keymaps in /usr/share/keymaps to choose from.
 keymap="us"
 
-# Should we first load the 'windowkeys' console keymap?  Most x86 users will
+# Should we first load the 'windowskeys' console keymap?  Most x86 users will
```
Comment (jsoref) [inline conf.d/keymaps:5]: I can't tell if these things are talking about `window`-`keys` or `windows`-`keys`.

File: conf.d/keymaps [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
 # say "yes" here.  Note that non-x86 users should leave it as "no".
 # Loading this keymap will enable VT switching (like ALT+Left/Right)
 # using the special windows keys on the linux console.
-windowkeys="NO"
+windowskeys="NO"
```
Comment (jsoref) [inline conf.d/keymaps:9]: It kinda feels like they mean this.

File: init.d/keymaps.in [hunk 3]
```diff
@@ -24,7 +24,7 @@ start()
 	: ${unicode:=$UNICODE}
 	: ${keymap:=$KEYMAP}
 	: ${extended_keymaps:=$EXTENDED_KEYMAPS}
-	: ${windowkeys:=$SET_WINDOWSKEYS}
+	: ${windowskeys:=$SET_WINDOWSKEYS}
```
Comment (jsoref) [inline init.d/keymaps.in:27]: This very much argues for `windows`

File: man/rc-sstat.8 [hunk 4]
```diff
@@ -20,7 +20,7 @@ info
 .Sh DESCRIPTION
 .Nm
 gathers and displays information about the status of services supervised
-by s6 then runs rc-status to show info about nnormal OpenRC services.
+by s6 then runs rc-status to show info about normal OpenRC services.
```
Comment (jsoref) [inline man/rc-sstat.8:23]: Sometimes random letters are intentional, but I don't _think_ that's the case here.

File: man/rc-update.8 [hunk 5]
```diff
@@ -35,7 +35,7 @@
 OpenRC uses named runlevels.  Rather than editing some obscure
 file or managing a directory of symlinks,
 .Nm
-exists to quickly add or delete services to and from from different runlevels.
+exists to quickly add or delete services to and from different runlevels.
```
Comment (jsoref) [inline man/rc-update.8:38]: Spell checkers have been flagging duplicate words for many decades, but I've only recently started to work with them. It's possible I've messed up in places, but I don't thinks so.

**Comment by jsoref** [comment]
I'm not at all attached to the signed-off-by lines, they're there because I regularly run into automation that screams at me when I don't have it. Quite happy to drop.

re: `window`-`keys`, what about this: https://github.com/OpenRC/openrc/pull/516#discussion_r851381294 -- is `$SET_WINDOWSKEYS` correct? Should it be `$SET_WINDOWKEYS`?

**Comment by williamh** [comment]
I would leave that the way it is on master for now.

Thanks,

William

**Comment by williamh** [comment]
The ChangeLog is auto generated so changes to it will be overwritten. I'm also thinking about removing that file in the future since I don't have a way to autogenerate it with the new build system. I will squash and merge all commits except that one.
Thanks much for doing this and for keeping all of your commits small so I could review them easily.

**Pull request closed without merging**